libraries/AP_Scripting/docs/docs.lua

```diff
@@ -56,168 +60,175 @@ function uint32_t_ud:toint() end
 ---@return number
 function uint64_t_ud:tofloat() end
 
--- Convert to integer, nil if too large to be represented by native int32
+-- Convert the uint64_t value to a standard Lua integer. Returns nil if the value is too large to be represented by a native 32-bit integer.
 ---@return integer|nil
 function uint64_t_ud:toint() end
 
--- Split into high and low half's, returning each as a uint32_t_ud
----@return uint32_t_ud -- high (value >> 32)
----@return uint32_t_ud -- low (value & 0xFFFFFFFF)
+-- Split the 64-bit value into high and low 32-bit halves, returning each as a uint32_t userdata.
+---@return uint32_t_ud -- The high 32 bits of the value (value >> 32).
+---@return uint32_t_ud -- The low 32 bits of the value (value & 0xFFFFFFFF).
 function uint64_t_ud:split() end
 
--- system time in milliseconds
----@return uint32_t_ud -- milliseconds
+-- Returns the system time in milliseconds since the autopilot booted.
+---@return uint32_t_ud -- Milliseconds since boot.
 function millis() end
 
--- system time in microseconds
----@return uint32_t_ud -- microseconds
+-- Returns the system time in microseconds since the autopilot booted.
+-- This should be stored in a uint64_t_ud to avoid precision loss.
+---@return uint64_t_ud -- Microseconds since boot.
 function micros() end
 
--- receive mission command from running mission
----@return uint32_t_ud|nil -- command start time milliseconds
----@return integer|nil -- command param 1
----@return number|nil -- command param 2
----@return number|nil -- command param 3
----@return number|nil -- command param 4
+-- Receives data from a currently running NAV_SCRIPT_TIME mission command.
+-- This function is the bridge between a mission file and a script, allowing for scripted maneuvers inside an auto mission.
+-- Returns nil for all values if no NAV_SCRIPT_TIME command is active.
+---@return uint32_t_ud|nil -- The time the NAV_SCRIPT_TIME command started, in milliseconds.
+---@return integer|nil -- The value from the mission command's 'command' (param 1) field.
+---@return number|nil -- The value from the mission command's 'param 2' field.
+---@return number|nil -- The value from the mission command's 'param 3' field.
+---@return number|nil -- The value from the mission command's 'param 4' field.
 function mission_receive() end
 
--- Print text, if MAVLink is available the value will be sent with debug severity
--- If no MAVLink the value will be sent over can
--- equivalent to gcs:send_text(7, text) or periph:can_printf(text)
----@param text string|number|integer
+-- Prints text to the GCS and connected peripherals.
+-- If a MAVLink GCS is connected, this is equivalent to `gcs:send_text(MAV_SEVERITY.DEBUG, text)`.
+-- If the script is running on a peripheral, the text will be sent over the CAN bus.
+---@param text string|number|integer -- The content to be printed. Non-string types will be converted to strings.
```

Comment:
I don't think we convert none strings? I might be worth testing, our print is not a nice as the built in print.